media-video/obs-studio/obs-studio-30.0.0-r1.ebuild

```diff
@@ -0,0 +1,294 @@
+	)
+	browser? (
+		|| (
+			>=app-accessibility/at-spi2-core-2.46.0:2
+			( app-accessibility/at-spi2-atk dev-libs/atk )
+		)
+		dev-cpp/nlohmann_json
+		dev-libs/expat
+		dev-libs/glib
+		dev-libs/nspr
+		dev-libs/nss
+		dev-libs/wayland
+		media-libs/alsa-lib
+		media-libs/fontconfig
+		media-libs/mesa[gbm(+)]
+		net-print/cups
+		x11-libs/cairo
+		x11-libs/libdrm
+		x11-libs/libXcursor
+		x11-libs/libXdamage
+		x11-libs/libXext
+		x11-libs/libXi
+		x11-libs/libxkbcommon
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXScrnSaver
+		x11-libs/libxshmfence
+		x11-libs/libXtst
+		x11-libs/pango
+	)
+	fdk? ( media-libs/fdk-aac:= )
+	jack? ( virtual/jack )
+	lua? ( ${LUA_DEPS} )
+	mpegts? (
+		net-libs/librist
+		net-libs/srt
+	)
+	pipewire? ( media-video/pipewire:= )
+	pulseaudio? ( media-libs/libpulse )
+	python? ( ${PYTHON_DEPS} )
+	qsv? ( media-libs/oneVPL )
+	speex? ( media-libs/speexdsp )
+	ssl? ( net-libs/mbedtls:= )
+	truetype? (
+		media-libs/fontconfig
+		media-libs/freetype
+	)
+	v4l? (
+		media-libs/libv4l
+		virtual/udev
+	)
+	vlc? ( media-video/vlc:= )
+	wayland? (
+		dev-libs/wayland
+		x11-libs/libxkbcommon
+	)
+	websocket? (
+		dev-cpp/asio
+		dev-cpp/nlohmann_json
+		dev-cpp/websocketpp
+		dev-libs/qr-code-generator
+	)
+"
+RDEPEND="${DEPEND}"
+
+QA_PREBUILT="
+	usr/lib*/obs-plugins/chrome-sandbox
+	usr/lib*/obs-plugins/libcef.so
+	usr/lib*/obs-plugins/libEGL.so
+	usr/lib*/obs-plugins/libGLESv2.so
+	usr/lib*/obs-plugins/libvk_swiftshader.so
+	usr/lib*/obs-plugins/libvulkan.so.1
+	usr/lib*/obs-plugins/swiftshader/libEGL.so
+	usr/lib*/obs-plugins/swiftshader/libGLESv2.so
+"
+
+pkg_setup() {
+	use lua && lua-single_pkg_setup
+	use python && python-single-r1_pkg_setup
+}
+
+src_unpack() {
+	default
+
+	if [[ ${PV} == 9999 ]]; then
+		git-r3_src_unpack
+		if use streamfx; then
+			cd ${P}/UI/frontend-plugins/ || die
+			git submodule add 'https://github.com/Xaymar/obs-StreamFX.git' streamfx || die
+			git submodule update --init --recursive || die
+		fi
+	else
+		rm -d ${P}/plugins/obs-browser || die
+		mv obs-browser-${OBS_BROWSER_COMMIT} ${P}/plugins/obs-browser || die
+
+		rm -d ${P}/plugins/obs-websocket || die
+		mv obs-websocket-${OBS_WEBSOCKET_COMMIT} ${P}/plugins/obs-websocket || die
+
+		if use streamfx; then
+			# need to be clone recursive - cmake error
+			git init || die # start dummy git repo to add submodule
+			cd ${P}/UI/frontend-plugins/ || die
+			git submodule add 'https://github.com/Xaymar/obs-StreamFX.git' streamfx || die
+			cd streamfx || die
+			git checkout ${STREAMFX_VERSION} || die
+			cd .. || die
+			git submodule update --init --recursive || die
+		fi
+	fi
+}
+
+src_prepare() {
+	default
+
+	sed -i '/-Werror$/d' "${WORKDIR}"/${P}/cmake/Modules/CompilerConfig.cmake || die
+
+	if use streamfx; then
+		echo 'add_subdirectory(streamfx)' &>>"${WORKDIR}"/${P}/UI/frontend-plugins/CMakeLists.txt || die
+	fi
+
+	if use amf; then
+		cd "${S}" || die
+		eapply "${FILESDIR}/${PN}-amf-${PV}.patch" || die
```

Comment:
the patch doesn't exist for obs 30

Reply:
Thanks for the catch I forget to change the patch name. I will update it tomorrow